certval/src/util/pdv_utilities.rs

```diff
@@ -7,10 +7,7 @@ use core::str::FromStr;
 
 use log::{debug, error};
 
-#[cfg(feature = "std")]
 use lazy_static::lazy_static;
-
-#[cfg(feature = "std")]
```

Comment:
I am not tracking this change and similar in this file. When I build with `cargo hack build --feature-powerset` it fails straightaway when building `cargo build --no-default-features`. I did not look at regex source code, but nothing in the docs indicates no-std support presently. How did you envision removing the feature gate here working?